releases/release-5.0.2.md

```diff
@@ -0,0 +1,121 @@
+    - Fix the bug that primary lock fallen back from async commit cannot be resolved. [#24397](https://github.com/pingcap/tidb/pull/24397)
+    - Statistics: fix a statistics GC problem that can cause duplicated fm-sketch records [#24357](https://github.com/pingcap/tidb/pull/24357)
+    - Avoid unnecessary pessimistic rollback when pessimistic locking receives ErrKeyExists [#23800](https://github.com/pingcap/tidb/pull/23800)
+
++ TiFlash
+
+    - Fix the potential concurrency problem when clone the shared delta index. [#2033](https://github.com/pingcap/tics/pull/2033)
+    - Fix the bug that incomplete data may make TiFlash fail to restart [#2003](https://github.com/pingcap/tics/pull/2003)
+    - Fix the problem that old dmfile is not removed atomically [#1925](https://github.com/pingcap/tics/pull/1925)
+    - Fix the issue that TiFlash may panic when `compaction filter` is enabled [#1891](https://github.com/pingcap/tics/pull/1891)
+    - Fix problem ExchangeSender will sometimes send duplicated data chunks [#1883](https://github.com/pingcap/tics/pull/1883)
+    - Fix problem that TiFlash can not resolve fallen back async commit lock [#1870](https://github.com/pingcap/tics/pull/1870)
+    - Fix unexpected timezone cast which may cause wrong result if the return column contains timestamp column [#1827](https://github.com/pingcap/tics/pull/1827)
+    - Fix a bug that TiFlash crash during Segment Split. [#1823](https://github.com/pingcap/tics/pull/1823)
+    - Fix bug that execution info for non root mpp task is not accurate enough [#1840](https://github.com/pingcap/tics/pull/1840)
+    - Fix the problem that obsolete data may not be removed when there is no more write to the storage [#1828](https://github.com/pingcap/tics/pull/1828)
+
++ Tools
+
+    - Backup & Restore (BR)
+
+        + Fix the bug that batch split region fails due to total key size exceeds raft entry limit [#1148](https://github.com/pingcap/br/pull/1148)
+
+## 以下 note 未分类。请将以下 note 进行分类 (New feature, Improvements, Bug fixes, Compatibility Changes 四类)，并移动到上面对应的标题下。如果某条 note 为多余的，请删除。如果漏抓取了 note，请手动补充
+
++ TiDB
+
+    - Update parser to fix lexer bug [#25015](https://github.com/pingcap/tidb/pull/25015)
+    - Fixed issue-24746, no longer allowing `INSERT INTO table PARTITION (<partitions>) ... ON DUPLICATE KEY UPDATE` to read from non-listed partitions [#25000](https://github.com/pingcap/tidb/pull/25000)
+    - Fix "index out of range" error when a SQL contains both group by and union. [#24551](https://github.com/pingcap/tidb/pull/24551)
+    - Fix wrong collation for concat function [#24301](https://github.com/pingcap/tidb/pull/24301)
+    - Fix global variable collation_server does not take effect in new session [#24156](https://github.com/pingcap/tidb/pull/24156)
+
++ TiKV
+
+    - release-note [#10278](https://github.com/tikv/tikv/pull/10278)
+    - No release notes [#10273](https://github.com/tikv/tikv/pull/10273)
```

Comment:
shall we remove "no release notes" items?

Reply:
No problem. If these PRs are really not for release notes, I'll remove them. If these PRs are for rn but no rn are added, please add the notes.

Reply:
Should remove. No release note for this item.

Reply:
Done